src/init/wazuh/wazuh.sh

```diff
@@ -223,4 +223,15 @@ WazuhUpgrade()
     if [ -f "$DIRECTORY/queue/alerts/execa" ]; then
         rm -f $DIRECTORY/queue/alerts/execa
     fi
+
+    # Update versions previous to Wazuh 4.8
```

Comment:
Nitpick? This function not only acts on versions previous to 4.8. ...How about something like 'Ensure vulnerability-detection and indexer are configured'?